apps/devhub/tests/test_views.py

```diff
@@ -2871,6 +2895,26 @@ def test_unlisted_addon_pass_validation(self, mock_sign_file):
         assert file_.status == amo.STATUS_LITE
         assert mock_sign_file.called
 
+    @mock.patch('devhub.views.sign_file')
+    def test_experiments_are_auto_signed(self, mock_sign_file):
+        """Experiment extensions (bug 1220097) are auto-signed."""
+        self.upload = self.get_upload('experiment.xpi')
+        self.addon.update(guid='experiment@xpi', is_listed=True,
+                          status=amo.STATUS_PUBLIC, trusted=False)
+        # We're going to sign even if it has errors/warnings.
+        self.upload.update(
+            validation='{"notices": 2, "errors": 1, "messages": [],'
+                       ' "metadata": {}, "warnings": 1,'
+                       ' "signing_summary": {"trivial": 1, "low": 0,'
+                       '                     "medium": 0, "high": 1},'
+                       ' "passed_auto_validation": 0}')
+        self.post()
+        assert self.addon.reload().status == amo.STATUS_PUBLIC
```

Comment:
Uploading a new version can change the status of the add-on automatically (using signals). We don't care about this change in this test thought, I'll remove it.